.travis.yml

```diff
@@ -21,6 +21,14 @@ env:
     - RAILS_ENV=test
     - CC_TEST_REPORTER_ID=f39e060a8b1a558ebd8aff75d5b9760bf1ae98f3f85d628ae28814f3c66438cd
     - DATABASE_URL=postgres://postgres@localhost/
+    - APP_PROTOCOL=http://
+    - APP_DOMAIN=localhost:3000
+    - TIMBER=dummy
+    - SEND_LOGS_TO_TIMBER=false
+    - SENDGRID_USERNAME_ACCEL=dummy
+    - SENDGRID_PASSWORD_ACCEL=dummy
+    - HEROKU_APP_URL=practicaldev.herokuapp.com
+    - SECRET_KEY_BASE=dummydummydummy
```

Comment:
I think it would be nice to document that these are env variables only required for the verify stage on Travis. I can see someone looking at this down the road and wondering why we have these or whether it
s safe to change or remove them.

Can we add a comment in the `travis.yml` file for our future selves?

Reply:
Can and will, was just waiting to see if we even go ahead with this approach at all. 😃 Thanks for the reminder though! 👍

Reply:
Makes sense! And thanks :)